pallets/allocations/src/tests.rs

```diff
@@ -267,4 +267,4 @@ fn no_issuance() {
 			Errors::BatchEmpty
 		);
 	})
-}
```

Comment:
I think this was added by `cargo fmt`, and I agree, I think we should have newlines at `EOF`